### PR TITLE
#63: fix for syntax highlighting when opening multiple files on comma…

### PR DESCRIPTION
**Pull request by ydeweerdt**
I don't know the details of all use cases, but removing the argdo resolves the issue for me.

### CODE REVIEW

**Comment by alberanid** [comment]
I confirm this pull request fixes the problem.

File: plugin/file_line.vim [hunk 1]
```diff
@@ -79,12 +79,14 @@ function! s:startup()
...
+		silent call s:handle_arg()
 		exec (argidx+1).'argument'
 		" Manually call Syntax autocommands, ignored by `:argdo`.
 		doautocmd Syntax
 		doautocmd FileType
 	endif
 endfunction
 
-autocmd VimEnter * call s:startup()
+if !isdirectory(expand("%:p"))
```
Comment (bogado) [inline plugin/file_line.vim]: Why are you doing this check?

Comment (ydeweerdt) [inline plugin/file_line.vim]: I do this because I don't want file-line to act on directory names, only on file names, I want to have NerdTree pop-up when specifying a directory.

Comment (jby) [inline plugin/file_line.vim]: If it's not related to the fix, maybe it should be in a separate PR?

**Comment by ydeweerdt** [comment]
It is indeed not related to the fix... I probably screwed up my very first
pull request...  I only wanted one commit to be pulled... Sorry...

**Comment by bogado** [comment]
No worries, the fix is fine, just remove the unnecessary check and I'll merge it

**Comment by ydeweerdt** [comment]
can you have a look if it is better now? I just reset my master branch to the single commit holding the fix for #63
